portfolio/src/main/webapp/comments.js

```diff
@@ -12,16 +12,81 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function loadComments(){
-  const maxComments = parseInt(document.getElementById("max-comments-select").value);
-  debugLog(maxComments);
-  fetch("/data" + "?max_comments=" + maxComments).then(response => response.json()).then((comments) => {
-    debugLog(comments);
-    const commentsList = document.getElementById('comments-container');
-    commentsList.innerHTML = '';
-    comments.forEach((comment) => {
-      commentsList.appendChild(createCommentElement(comment.email, comment.comment, comment.date));
-    });
+function loadPage(pgNumber = 1){
```

Comment:
loadPage() sounds like you might be loading the whole HTML page, so changing this to loadCommentsPage() probably makes sense.